tests/test_builtin_casters.py

```diff
@@ -297,7 +300,8 @@ def cant_convert(v):
 
     assert convert(7) == 7
     assert noconvert(7) == 7
-    cant_convert(3.14159)
+    assert convert(3.14159) == 3
```

Comment:
These deprecation warnings are concerning

Reply:
Ok I extracted out this logic into a helper fixture.